src/scipp/core/reduction.py

```diff
@@ -190,17 +276,31 @@ def nanmax(x: _cpp.Variable,
            dim: Optional[str] = None,
            *,
            out: Optional[_cpp.Variable] = None) -> _cpp.Variable:
-    """Element-wise max ignoring not a number values over the specified
-    dimension or all dimensions if not provided.
-
-    :param x: Input data.
-    :param dim: Optional dimension along which to calculate the max. If not
-                given, the max over all dimensions is calculated.
-    :param out: Optional output buffer.
-    :raises: If the dimension does not exist, or the dtype cannot be summed,
```

Comment:
Missing `raises`? Although many of these seem relatively pointless (and also wrong, they all say 'summed')